.cci.jenkinsfile

```diff
@@ -34,3 +62,51 @@ codestyle: {
   }
 }
 }
+
+// Build FCOS and do a kola basic run
+stage("More builds and test") {
+parallel fcos: {
+  coreos.pod(image: COSA_IMAGE, runAsUser: 0, kvm: true, memory: "2048Mi", cpu: "2") {
+    stage("Build FCOS") {
+      checkout scm
+      unstash 'build'
+      sh """
+        set -xeuo pipefail
+        mkdir insttree
+        tar -C insttree -xzvf insttree.tar.gz
+        rsync -rlv insttree/ /
+        coreos-assembler init --force https://github.com/coreos/fedora-coreos-config
+        mkdir -p overrides/rootfs
+        mv insttree/* overrides/rootfs/
+        rmdir insttree
+        coreos-assembler build
+      """
+    }
+    stage("FCOS basic") {
+      timeout(time: 30, unit: 'MINUTES') {
+        sh """
+          set -xeuo pipefail
+          cosa kola run --basic-qemu-scenarios
+        """
+      }
+    }
+  }
+},
+buildopts: {
+  coreos.pod(image: COSA_BUILDROOT_IMAGE, runAsUser: 0) {
+      checkout scm
+      sh """
+        set -xeuo pipefail
+        git submodule update --init
+
+        git clean -dfx && git submodule foreach git clean -dfx
```

Comment:
Hmm, we don't need this, right? Each pod should get a fresh workspace.

Reply:
Yeah, but I'm running multiple builds here.  I could split it into a separate pod but...some worry about the cost of repeated source cloning.

Reply:
I don't follow; this branch allocates its own pod already, no?

Reply:
You're right that the first clean is unnecessary, but I was trying to be defensive when the section is copy-pasted.  However, I switched this now to use a `git worktree` which is actually nicer anyways.

Reply:
Ahh gotcha, I somehow missed the copy/paste pattern you were going for. But yeah, `git worktree` is cleaner.